modules/oadp-restic-restore-failing-psa-policy.adoc

```diff
@@ -0,0 +1,49 @@
+```
+
+.Solution
+
+**Restore SCC before Pod**: Set the restore priority field on the Velero server to restore SCC before Pod.
+
+.Example DPA CR
+[source,yaml]
+----
+oc get dpa -o yaml
+configuration:
+  restic:
+    enable: true
+  velero:
+    args:
+      restore-resource-priorities: 'securitycontextconstraints,customresourcedefinitions,namespaces,storageclasses,volumesnapshotclass.snapshot.storage.k8s.io,volumesnapshotcontents.snapshot.storage.k8s.io,volumesnapshots.snapshot.storage.k8s.io,datauploads.velero.io,persistentvolumes,persistentvolumeclaims,serviceaccounts,secrets,configmaps,limitranges,pods,replicasets.apps,clusterclasses.cluster.x-k8s.io,endpoints,services,-,clusterbootstraps.run.tanzu.vmware.com,clusters.cluster.x-k8s.io,clusterresourcesets.addons.cluster.x-k8s.io'<1>
+    defaultPlugins:
+    - gcp
+    - openshift
+----
+<1> If you have an existing restore resource priority list, ensure you combine that existing list with the complete list.
+
+
+If the application is not aligned with security standards, an error can occur regardless of the SCC. 
+
+Ensure that the security standards for the application pods are aligned, as provided in link:https://access.redhat.com/solutions/7002730[Fixing PodSecurity Admission warnings for deployments], to prevent deployment warnings.
```

Comment:
This was a bit confusing to parse at first, but I think I get what it's saying now... Can this be more of a procedure (and drop the bold markup) to maybe help with clarity? Something like:
```suggestion
. In your DPA custom resource (CR), check or set the `restore-resource-priorities` field on the Velero server to ensure that `securitycontextconstraints` is listed in order before `pods` in the list of resources:
+
[source,terminal]
----
$ oc get dpa -o yaml
----
+
.Example DPA CR
[source,yaml]
----
# ... 
configuration:
  restic:
    enable: true
  velero:
    args:
      restore-resource-priorities: 'securitycontextconstraints,customresourcedefinitions,namespaces,storageclasses,volumesnapshotclass.snapshot.storage.k8s.io,volumesnapshotcontents.snapshot.storage.k8s.io,volumesnapshots.snapshot.storage.k8s.io,datauploads.velero.io,persistentvolumes,persistentvolumeclaims,serviceaccounts,secrets,configmaps,limitranges,pods,replicasets.apps,clusterclasses.cluster.x-k8s.io,endpoints,services,-,clusterbootstraps.run.tanzu.vmware.com,clusters.cluster.x-k8s.io,clusterresourcesets.addons.cluster.x-k8s.io' <1>
    defaultPlugins:
    - gcp
    - openshift
----
<1> If you have an existing restore resource priority list, ensure you combine that existing list with the complete list.

. Ensure that the security standards for the application pods are aligned, as provided in link:https://access.redhat.com/solutions/7002730[Fixing PodSecurity Admission warnings for deployments], to prevent deployment warnings. If the application is not aligned with security standards, an error can occur regardless of the SCC. 
```